content/cloud-computing/concepts/paas/paas.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'PaaS'
+Description: 'Platform as a service (PaaS): a service model that offers developers a platform to create, manage, and deploy their applications.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Cloud'
+  - 'Platform'
+  - 'Development'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Platform as a Service
+
+**Platform as a service (PaaS)** is a service model offered by cloud service providers that allows developers to develop and deploy apps without worrying about the underlying software and hardware their applications depend upon. When using a PaaS, the service provider provides and maintains the following:
```

Comment:
First sentence goes for too long. Platform as a Service is a service model offered to simplify the process of developing and deploying apps. With PaaS, developers don't need to worry about the underlying...